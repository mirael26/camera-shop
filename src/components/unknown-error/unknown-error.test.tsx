import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import UnknownError from './unknown-error';

test('UnknownError component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<UnknownError/>);
  expect(asFragment()).toMatchSnapshot();
});
