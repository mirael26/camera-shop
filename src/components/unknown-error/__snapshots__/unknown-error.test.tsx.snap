// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnknownError component renders correctly 1`] = `
<DocumentFragment>
  <section
    class="unknown-error"
    data-testid="unknown-error"
  >
    <div
      class="container"
    >
      <h1
        class="title title--h2 unknown-error__title"
      >
        Что-то пошло не так :(
      </h1>
      <a
        class="btn btn--purple"
        href="/"
      >
        Вернутся на главную
      </a>
    </div>
  </section>
</DocumentFragment>
`;
